core/subsurface-qt/diveobjecthelper.cpp

```diff
@@ -249,6 +249,28 @@ QStringList getFullCylinderList()
 	return cylinders;
 }
 
+QString formatSalinity(const dive *d)
+{
+	int salinity = get_dive_salinity(d);
+	if (!salinity)
+		return QString();
+        return QString("%1g/ℓ").arg(salinity / 10.0);
```

Comment:
I reckon this should be localized, i.e. "%L1g/ℓ"?

BTW: Here and elsewhere, indentation is with spaces instead of TABs.

Reply:
Yeah - probably should be localized - that might take a bit to sort out, but I'll see what I can figure out.

Reply:
It should be trivial: The only thing you have to do is place an `L` between `%` and `1` (I think).

Reply:
```
return gettextFromC::tr("%1g/ℓ").arg(salinity / 10.0);
```

should work.

Reply:
Miscommunication: I'm wasn't talking about localization of the string, but the value. I.e. comma or period for decimal separator. That is automatically done if you add a capital `L`.

[That said: do we support imperial units for water density?]

Reply:
aha! I can definitely put in the ``L`` for the comma/period. And good - because I didn't want to figure out imperial units for water density anyway!